__tests__/app.test.js

```diff
@@ -107,6 +86,66 @@ describe("GET /api/users", () => {
       })
   })
   test("throws an error if route is wrong", () => {
-      return request(app).get("/api/banana").expect(404);
+      return request(app).get("/api/banana").expect(404)
+    })
+})
+
+describe("PATCH /api/articles/:article_id", () => {
+  test('response with an updated inc_vote in the updated article', () => {
+    return request(app)
+    .patch("/api/articles/11")
+    .send({ inc_vote: 1 })
+    .expect(201)
+    .then(({body}) => {
+      expect(body.updatedArticle).toEqual(
+        expect.objectContaining({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number)
+        })
+      )
+      expect(body.updatedArticle.votes).toBe(1);
+      expect(body.updatedArticle.article_id).toBe(11);
```

Comment:
You could have changed the test to a specific number on line 102 and 108 but this will work too